tests/pr_test.go

```diff
@@ -87,7 +87,7 @@ func TestRunCompleteUpgradeExample(t *testing.T) {
 		ResourceGroup:      resourceGroup,
 		BestRegionYAMLPath: regionSelectionPath,
 		TerraformVars: map[string]interface{}{
-			"etcd_version": "3.3", // should always test the lowest available version of etcd that the module supports
+			"etcd_version": "3.4", // should always test the lowest available version of etcd that the module supports
```

Comment:
I have made the changes to include lowest version. This was done because of the [review comment](https://github.com/terraform-ibm-modules/terraform-ibm-icd-etcd/pull/70#discussion_r1414480663)